layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-bb.org_%.bbappend

```diff
@@ -2,23 +2,65 @@ FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
 
 inherit resin-u-boot
 
-SRC_URI += " \
+UBOOT_MACHINE:beagleplay = "am62x_beagleplay_a53_defconfig"
+UBOOT_MACHINE:beagleplay-k3r5 = "am62x_beagleplay_r5_defconfig"
+
+# No need to use the UBOOT_CONFIG_FRAGMENTS from the meta-ti layer for the beagplay,
+# since the 2024.04-BeaglePlay branch already has the necessary changes.
+UBOOT_CONFIG_FRAGMENTS:beagleplay = ""
+UBOOT_CONFIG_FRAGMENTS:beagleplay-k3r5 = ""
+
+SRC_URI:append:beaglebone-ai64-k3r5 = " \
   file://integrate_with_balena_u-boot-environment.patch \
   file://balenaos_bootcommand.cfg \
-  file://0005-env_default.h-Add-support-for-OS_KERNEL_CMDLINE.patch \
```

Comment:
Why was this moved to the bottom? If for no reason, let's have it where it used to be, in this way there will be less changes in the PR and easier for review

Reply:
Does not look like it was moved back to the original place.

Reply:
Done now (finally)... Please confirm and resolve as needed.

Reply:
Not resolved